modules/minimum-required-permissions-gcp-upi.adoc

```diff
@@ -0,0 +1,307 @@
+* `deploymentmanager.deployments.delete`
+* `deploymentmanager.deployments.get`
+* `deploymentmanager.deployments.list`
+* `deploymentmanager.manifests.get`
+* `deploymentmanager.operations.get`
+* `deploymentmanager.resources.list`
+====
+
+.Required Service Account permissions
+[%collapsible]
+====
+* `iam.serviceAccountKeys.create`
+* `iam.serviceAccountKeys.delete`
+* `iam.serviceAccountKeys.get`
+* `iam.serviceAccountKeys.list`
+* `iam.serviceAccounts.actAs`
+* `iam.serviceAccounts.create`
+* `iam.serviceAccounts.delete`
+* `iam.serviceAccounts.get`
+* `iam.serviceAccounts.list`
+* `resourcemanager.projects.get`
+* `resourcemanager.projects.getIamPolicy`
+* `resourcemanager.projects.setIamPolicy`
+====
+
+.Required permissions for deleting network resources
+[%collapsible]
+====
+* `compute.addresses.delete`
+* `compute.addresses.deleteInternal`
+* `compute.addresses.list`
+* `compute.firewalls.delete`
+* `compute.firewalls.list`
+* `compute.forwardingRules.delete`
+* `compute.forwardingRules.list`
+* `compute.networks.delete`
+* `compute.networks.list`
+* `compute.networks.updatePolicy`
+* `compute.routers.delete`
+* `compute.routers.list`
+* `compute.routes.list`
+* `compute.subnetworks.delete`
+* `compute.subnetworks.list`
+====
+
+.Required permissions for deleting load balancer resources
+[%collapsible]
+====
+* `compute.regionBackendServices.delete`
+* `compute.regionBackendServices.list`
+* `compute.targetPools.delete`
+* `compute.targetPools.list`
+====
+
+.Required permissions for deleting DNS resources
+[%collapsible]
+====
+* `dns.changes.create`
+* `dns.managedZones.delete`
+* `dns.managedZones.get`
+* `dns.managedZones.list`
+* `dns.resourceRecordSets.delete`
+* `dns.resourceRecordSets.list`
+====
+
+.Required permissions for deleting Service Account resources
+[%collapsible]
+====
+* `iam.serviceAccounts.delete`
+* `iam.serviceAccounts.get`
+* `iam.serviceAccounts.list`
+* `resourcemanager.projects.getIamPolicy`
+* `resourcemanager.projects.setIamPolicy`
+====
+
+.Required permissions for deleting compute resources
+[%collapsible]
+====
+* `compute.disks.delete`
+* `compute.disks.list`
+* `compute.instanceGroups.delete`
+* `compute.instanceGroups.list`
+* `compute.instances.delete`
+* `compute.instances.list`
+* `compute.instances.stop`
+* `compute.machineTypes.list`
+====
+
+.Required for deleting storage resources
+[%collapsible]
+====
+* `storage.buckets.delete`
+* `storage.buckets.getIamPolicy`
+* `storage.buckets.list`
+* `storage.objects.delete`
+* `storage.objects.list`
+====
+
+.Required permissions for deleting health check resources
+[%collapsible]
+====
+* `compute.healthChecks.delete`
+* `compute.healthChecks.list`
+* `compute.httpHealthChecks.delete`
+* `compute.httpHealthChecks.list`
+====
+
+.Additional Images permissions for installation
+[%collapsible]
+====
+* `compute.images.list`
+====
+
+.Required permissions for deleting image resources
+[%collapsible]
+====
+* `compute.images.delete`
+====
+
```

Comment:
Suggest to add another section, for example, 

Required permissions for deleting Deployment Manager permissions
- deploymentmanager.deployments.delete
- deploymentmanager.deployments.list
- deploymentmanager.operations.get

Reply:
Addressed